docs/Deployment.md

```diff
@@ -1,270 +1,123 @@
-After=network.target mysql.service
-
-[Service]
-
-######################################################
-##
-##      NODEJS SERVER CONFIGURATION
-##
-
-# Port where the server will listen
-# With '%i' we can launch app with `systemctl start kapp@3000`
-Environment=PORT=%i
-
-# Do not serve directly internet
-Environment=HOSTNAME=localhost
-
-#  Database configuration
-#       Default can be found in the /server/config/ folder of the project.
-
-# MySQL address
-#Environment=DB_HOST=myexternaldatabase.com:4245
-
-# Login user/password
-Environment=DB_USER=kapp
-Environment=DB_PWD=ComplicatedPassword
-
-# Database to use
-#   Warning: if define, backup config must also be changed.
-#Environment=DB_DATABASE=my_awesome_database
-
-
-# JWT Secret
-Environment=JWT_SECRET=UltraComplicatedSecret
-
-
-### SERVER CONFIGURATION
-
-# Launch
-
-Type=simple
-WorkingDirectory=/srv/kapp/
-ExecStart=/usr/bin/node server/index.js
-Restart=on-failure
-RestartSec=10
-
-# Logging
-StandardOutput=syslog
-StandardError=syslog
-
-# Security
-
-DynamicUser=yes
-
-# 'CAP_NET_BIND_SERVICE' if PORT is less than 1024
-CapabilityBoundingSet=
-NoNewPrivileges=yes
-ProtectControlGroups=yes
-ProtectKernelModules=yes
-
-[Install]
-WantedBy=multi-user.target
-```
-
+## Process
 
-### Proxying / Load balancing
+The application work in a continuous deployment system, 
+where everything put on master is deployed directly on a _staging_ environment 
+and when a new release is made through Github, the server automatically update it-self.
 
-There are several tools that can be used as proxies.
-Why add a proxy in front of the _NodeJS_ server?
-- **Load balancing**: NodeJS run in a single threaded environment, 
-if you want to use all the available CPU cores of your machine, 
-you can launch multiple instance of the application and let the load balancer 
-share the load between instances.
+### The server
 
-- **Caching, Compression**: The proxy can send the front app files 
-to clients, with compression and cache control.
+In a fully configured server, there are multiple things working together:
 
-- **Security**: By default, the NodeJS ***does not*** use _HTTPS_. 
-It can be configured in `/server/config/web.js`, but not by environment variables.
-Instead, you can add HTTPS directly in the proxy 
-(n.b.: data between _Proxy_ and _NodeJS_ ***will not*** be encrypted). 
+#### systemd
 
-#### Caddy
+Systemd is the service manager in Linux. 
+It will handle for us _auto-restart_ in case of failure and on reboot.
+It will also provide a secure environment for the application.
 
-[Caddy](https://caddyserver.com/) is a really good web server, with automatic HTTPS, 
-and an easy configuration.
+_Config files are located at_:
+- `/etc/systemd/system/<name>@.service`
+- `/etc/systemd/system/<name>-backup.service`
+- `/etc/systemd/system/<name>-backup.timer`
 
-First, install _Caddy_ : `curl https://getcaddy.com | bash -s personal http.cache`.
+`<name>`: `kapp` or `kapp-staging` (e.g.)
```

Comment:
I don't understand this example... Maybe i need to be more familiar with systemd

Reply:
It's not an example, just an explanation for the `<name>` placeholder